test/Conversion/tritongpu_to_llvm_blackwell.mlir

```diff
@@ -384,34 +384,28 @@ module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 4 : i32} {
 
 // -----
 
-#blocked = #ttg.blocked<{sizePerThread = [1, 1], threadsPerWarp = [1, 32], warpsPerCTA = [1, 4], order = [1, 0]}>
-#blocked1 = #ttg.blocked<{sizePerThread = [1, 256], threadsPerWarp = [32, 1], warpsPerCTA = [4, 1], order = [0, 1]}>
+#blocked = #ttg.blocked<{sizePerThread = [1, 256], threadsPerWarp = [32, 1], warpsPerCTA = [4, 1], order = [0, 1]}>
 #tmem = #ttng.tensor_memory_encoding<blockM = 128, blockN = 256, unpacked = true>
 
 module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 4 : i32, ttg.shared = 65544 : i32, ttg.target = "cuda:100", ttg.tensor_memory_size = 128 : i32, "ttg.threads-per-warp" = 32 : i32} {
   // CHECK-LABEL: @tensor_memory_ld_128x256
-  // CHECK: tcgen05.st.sync.aligned.32x32b.x64.b32
-  // CHECK: tcgen05.st.sync.aligned.32x32b.x64.b32
-  // CHECK: tcgen05.st.sync.aligned.32x32b.x64.b32
-  // CHECK: tcgen05.st.sync.aligned.32x32b.x64.b32
+  // CHECK-COUNT-2: tcgen05.st.sync.aligned.32x32b.x128.b32
+  // CHECK-NOT: tcgen05.st
   // CHECK: tcgen05.wait::st.sync.aligned
-  // CHECK: tcgen05.ld.sync.aligned.32x32b.x64.b32
-  // CHECK: tcgen05.ld.sync.aligned.32x32b.x64.b32
-  // CHECK: tcgen05.ld.sync.aligned.32x32b.x64.b32
-  // CHECK: tcgen05.ld.sync.aligned.32x32b.x64.b32
+  // CHECK-COUNT-2: tcgen05.ld.sync.aligned.32x32b.x128.b32
+  // CHECK-NOT: tcgen05.ld
```

Comment:
>The goal of the heuristic here is to never use more than half of the available registers for a TMEM instruction, since each instruction requires all registers to be live at once.

That's actually was not the goal of the heuristic, instead it was

>The goal of the heuristic here is to always less than half of the available registers for a TMEM instruction, since each instruction requires all registers to be live at once.



It ensures that when a workload will require 256 registers in total, we should not use the largest tmem message size (x128) as two of those back to back can cause register spilling. This PR now re-enables that behavior?

Reply:
I didn't observe any problems in my benchmarking with this, but I can change that back no problem.

Reply:
Fixed in https://github.com/triton-lang/triton/pull/6692

Reply:
Previously I observed spilling by checking the ptxas logs when running tutorial 10. I would be surprised if it didn't still happen but thank you for your quick PRs nonetheless :)